Docs/Ai_help.md

```diff
@@ -0,0 +1,18 @@
+# AI Section
+
+## AI Help
+
+In this section we will go over the AI help that has been through out the process of creating the project. This will be about what AI help has been used and to what extend.
+
+### ChatGBT
+
+ChatGBT is one of the AI's that we have used through out the project for smaller questions. Theses questions range from C# code, refactoring of code, setup of docker or .NET projects. With the help from the AI we can ask questions and get quick response that can help us in the right direction. There are somethings to consider when using ChatGBT the main thing will be the validity of the response since we can't guarantee that it's right.
```

Comment:
Can we perphaps elaborate on what type of c# code, we have used it for?

Is it things like using APS.NET Core primitives to attach handlers call in Program.cs.
How to write a certain pattern?
How to use something with the page model in razor pages?
Or something fourth?